src/core/reducers/installations.js

```diff
@@ -13,10 +14,12 @@ import {
   UNINSTALL_COMPLETE,
   acceptedInstallTypes,
 } from 'core/constants';
+import log from 'core/logger';
 
 
 export default function installations(state = {}, { type, payload }) {
   if (!acceptedInstallTypes.includes(type)) {
+    log.info(`${type} not in acceptedInstallTypes so action is being ignored.`);
```

Comment:
Well what's weird is in most reducers we just check for a bunch of action `type`s (with either an `if/else` or `switch` control), act on them, and `return state;` if none of the types match.

Here we return early if something isn't in a constant array when we have to define the types we act on anyway. It seems like we're checking twice here and it's very different to the rest of our code.